src/api/resolvers/types/mutations/inputs/project/GraphQLAddComponentToProjectInput.ts

```diff
@@ -0,0 +1,22 @@
+import { GraphQLInputObjectTypeConfig, GraphQLNonNull, GraphQLString, GraphQLList, GraphQLID, GraphQLInputObjectType } from "graphql";
+
+const addComponentToProjectInputConfig: GraphQLInputObjectTypeConfig = {
+    name: "AddComponentToProjectInput",
+    description: "The inputs for the addComponentToProject mutation",
+    fields: () => ({
+        clientMutationID: {
+            type: GraphQLString,
+            description: "An arbitraty string to return together with the mutation result"
+        },
+        projectId: {
+            type: GraphQLNonNull(GraphQLID),
+            description: "The id of the project where to add the component"
+        },
+        componentId: {
+            type: GraphQLNonNull(GraphQLID),
+            description: "The id of the component to remove from the project"
```

Comment:
```suggestion
            description: "The id of the component to add to the project"
```
WTF?! Confusing add and remove?